content/en/docs/setup/install/operator/index.md

```diff
@@ -192,7 +192,7 @@ You can show the differences in the generated manifests between the default prof
 
 {{< text bash >}}
 $ istioctl manifest generate > 1.yaml
-$ istioctl manifest generate -f samples/pilot-k8s.yaml > 2.yaml
+$ istioctl manifest generate -f samples/operator/pilot-k8s.yaml > 2.yaml
```

Comment:
Use @ so the files are clickable links in the doc.

```suggestion
$ istioctl manifest generate -f @samples/operator/pilot-k8s.yaml@ > 2.yaml
```

Reply:
@frankbu  They are not clickable. Those files do not exist in istio repo, they are from the operator repo which are included in the release tarball.

Reply:
This brings up an interesting question. You can reference other repos in the links as follows (https://istio.io/about/contribute/creating-and-editing-pages/#links-to-github-files):
```
{{< text syntax="bash" repo="operator" >}}
$ cat @README.md@
{{< /text >}}
```

But the challenge here is that the path in github.com/istio/operator is different than the path in the archive. Should these files be located in github.com/istio/operator/samples/operators for consistency? Seems hacky, but would fix the link. Would be nice if there was a way to specify a different GitHub path, but I don't think we have that functionality right now.

Reply:
Correct, we don't have the ability to point to some random location in the target repo.

Reply:
Merging sample directories from multiple repos into one samples directory in the release image seems like a bad idea. I suppose if we only do it in very specific cases, there's no risk of collision, but in the general case it does seem risky.